### ISSUE TITLE
VU meter display problem

### ISSUE THREAD

**Issue by DroidU** [schismtracker/schismtracker#586]
The VU meter does not show anything in the surround channel. If I set the playback to mono, the VU meter works fine. This phenomenon is probably due to the mono sample inverse phase.

Related request: If you have to touch the VU meter, it should be possible to configure the falloff speed/rate of the VU meter on System Configuration (Ctrl-F1) page.

Another thing that is not perfect in the original Impulse Tracker:
If I reverse the Left/Right Output with Alt+R, it only displays the fact of the reversed. However, I do not see what state the output is in.
Here is a suggestion: If I press Alt+R for the first time, then as now, display "Left/right outputs reversed". If I do it again, then "Left/right outputs restored" should be displayed.

**Comment by mrpapersonic** [comment]
yep, the current code gets the average of maximum output of the two channels and then gets the absolute value, when it should get the absolute value and then average those, thanks!
